### DISCUSSION TITLE
Hello old friend !

### DISCUSSION THREAD

**Discussion by 0x07CB** [ohmysh/ohmysh#32] [General]
Hi, I hope to be not .. to late.
In past I have work with you and I have feel fun to code on an same repository with you.
I think you code more fast like the past I remember, I have not forgotten the speed you can deploy to fix issues and resolve problems, from errors in my code to every pull requests to your repository.

Now I want know your plan and the next, what code you need to write and what you have less time to do, I'm ready to see if I can work again with you.

Have a nice day.
Please never stop to learn coding, you are good.

TTYL.
- Rick Sanchez

**Comment by langonginc** [comment]
OMG, I don't know this function in GitHub, and I'm so sorry that I see this after a few months. 

I don't have any idea right now. If you have, please tell me. 

Have a nice day!
